src/pages/Home.jsx

```diff
@@ -50,24 +54,54 @@ const AppBar = styled(MuiAppBar, {
       easing: theme.transitions.easing.easeOut,
       duration: theme.transitions.duration.enteringScreen,
     }),
-    background: "white",
+    background: theme.palette.background.default,
     boxShadow: "none",
   }),
 }));
 
 export default function Home() {
   const [open, setOpen] = useState(true);
+  const [version, setVersion] = useState("???");
+
+  const [apiKeyDialogOpen, setApiKeyDialogOpen] = useState(false);
+  const { client: qdrantClient } = useClient();
+
+  async function getQdrantVersion() {
+    // console.log("qdrantClient.api", qdrantClient._openApiClient);
+    try {
+      let telemetry = await qdrantClient.api("service").telemetry();
+      setVersion(telemetry.data.result.app.version);
+    } catch (error) {
+      if (error.status === 403) {
```

Comment:
Maybe we can be a bit more strict and match on the body content as well. We can probably just match on "Invalid api-key".

https://github.com/qdrant/qdrant/blob/f1d2490c2085bad952ba43cd40ee652fb401db7b/src/actix/api_key.rs#L69

Reply:
that's implementation-specific. Cloud & native auth might have different messages